Regression/Checksum/benchmarks_json/collisionXYZ.json

```diff
@@ -6,25 +6,25 @@
     "Ex": 0.0,
     "Ey": 0.0,
     "Ez": 0.0,
-    "T_electron": 351982.0169218243,
-    "T_ion": 349599.6939052666
+    "T_electron": 381957.7394223898,
+    "T_ion": 319565.6269784763
   },
```

Comment:
The large difference in these temperatures is a bit concerning. I may need to inspect this test further.

Reply:
Looks like this relatively large changed happened on PR #5110.